tensorflow/core/util/mkl_util.h

```diff
@@ -130,6 +130,17 @@ enum class MklQuantization {
 
 static const int kSmallBatchSize = 32;
 
+#ifdef ENABLE_MKLDNN_V1
+inline void execute_primitives(
+    std::vector<mkldnn::primitive>& primitives, std::shared_ptr<stream> stream,
+    std::vector<std::unordered_map<int, memory>>& net_args) {
+  DCHECK_EQ(primitives.size(), net_args.size());
+  for (size_t i = 0; i < primitives.size(); ++i) {
+    primitives.at(i).execute(*stream, net_args.at(i));
+  }
+}
+
+#endif  // ENABLE_MKLDNN_V1
```

Comment:
This can be merged with the `#ifdef ENABLE_MKLDNN_V1` below.

Reply:
@penpornk Done, Thanks for this great cache.